### PR TITLE
Add acceptance tests of the demo application, document the workflows

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/create_application.rst [hunk 1]
```diff
@@ -294,7 +294,59 @@ Then follow the sections in the install application guide:
...
+
+In the `Makefile` you should configure the dump file as `DUMP_FILE`.
+
+In the file `.github/workflows/main.yaml` you should uncomment all the lines related to the
+acceptance tests.
+
+The included acceptance tests will do the following tests:
+- The base application page (Get status code 200).
+- The `dynamic.json` for the desktop (Get status code 200).
+- The checkers level one.
```
Comment (sbrunner) [inline doc/integrator/create_application.rst:351]: Update

File: doc/integrator/create_application.rst [hunk 2]
```diff
@@ -294,7 +294,59 @@ Then follow the sections in the install application guide:
...
+................
+
+To have some acceptance tests you need to have a minimal dump of your application in the repository,
+it can be obtained with:
+
+.. prompt:: bash
+
+    scripts/db-backup --arg=--schema=<schema> ../dump.backup
+
+In the `Makefile` you should configure the dump file as `DUMP_FILE`.
```
Comment (sbrunner) [inline doc/integrator/create_application.rst:343]: personalize the db.restore ...

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/Makefile [hunk 3]
```diff
@@ -34,3 +35,34 @@ eslint: ## Runs the eslint checks
 		eslint $(find {{cookiecutter.package}} -type f -name '*.js' -print 2> /dev/null)
 	docker-compose run --entrypoint= --no-deps --rm --volume=$(pwd)/geoportal:/app geoportal \
 		eslint $(find {{cookiecutter.package}} -type f -name '*.ts' -print 2> /dev/null)
+
+.PHONY: qgis
+qgis: ## Run QGIS desktop
+	docker-compose -f docker-compose.yaml -f docker-compose-qgis.yaml run --rm qgis
+
+secrets.tar.bz2.gpg: env.secrets ## Create the secrets.tar.bz2.gpg files we depends on
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/Makefile:43]:
```suggestion
secrets.tar.bz2.gpg: env.secrets ## Encrypt the secrets for committing changes
```
Note that the filename is already the target name, not sure we need to repeat it in description.

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/Makefile [hunk 4]
```diff
@@ -34,3 +35,34 @@ eslint: ## Runs the eslint checks
...
+	tar -jcf secrets.tar.bz2 $^
+	rm -f $@
+	gpg --symmetric --cipher-algo AES256 --batch \
+		--passphrase=$(shell gopass show gs/ci/large-secret-passphrase) secrets.tar.bz2
+	rm secrets.tar.bz2
+
+.PHONY: secrets
+secrets: ## Decript the sectrets.tar.bz2.gpg file
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/Makefile:51]:
```suggestion
secrets: ## Decrypt the secrets.tar.bz2.gpg file
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/docker-compose-lib.yaml [hunk 5]
```diff
@@ -1,4 +1,5 @@
----
+# This files is managed by c2cgeoportal, he contains the default services configuration
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/docker-compose-lib.yaml:1]:
```suggestion
# This file is managed by c2cgeoportal, it contains the default services configuration
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_create_template/tests/test_testapp.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+import re
+
+import polib
+import pytest
+import requests
+
+
+@pytest.mark.parametrize("test_number", [0, 1])
+def test_po(test_number: int) -> None:
+    """Tests that the generated pot files are identical between the command lin and the view."""
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_create_template/tests/test_testapp.py:12]:
```suggestion
    """Tests that the generated pot files are identical between the command line and the view."""
```

**Review by arnaud-morvan — APPROVED** [review]
LGTM, minor comments and typos

**Pull request merged**